compiler/rustc_middle/src/mir/syntax.rs

```diff
@@ -1444,6 +1444,8 @@ pub enum BinOp {
     Ge,
     /// The `>` operator (greater than)
     Gt,
+    /// The `<=>` operator (three-way comparison, like `Ord::cmp`)
```

Comment:
This should say which values can be returned. I assume the return type is the same as the input type? But no that doesn't work when comparing unsigned integers.

This files serves as our de-facto "MIR LangRef".